core/common/boot/src/main/java/org/eclipse/edc/boot/system/ExtensionLoader.java

```diff
@@ -127,4 +127,21 @@ public <T> List<T> loadExtensions(Class<T> type, boolean required) {
         return serviceLocator.loadImplementors(type, required);
     }
 
+    /**
+     * Parses the ConsoleMonitor log level from the program args. If no log level is provided, defaults to Level default.
+     */
+    private static ConsoleMonitor.Level parseLogLevel(String[] programArgs) {
+        return Set.of(programArgs).stream()
+                .filter(arg -> arg.startsWith(ConsoleMonitor.LEVEL_PROG_ARG))
+                .map(arg -> {
+                    try {
+                        return ConsoleMonitor.Level.valueOf(arg.split("=")[1]);
+                    } catch (IllegalArgumentException e) {
+                        throw new IllegalArgumentException(String.format("Illegal Console Monitor log level value: %s. Possible values are DEBUG, INFO, WARNING, SEVERE", arg.split("=")[1]));
```

Comment:
the "possible values" can be obtained from the `Level` enum, not that they will easily change or anything, but looks like the more natural thing to do instead of put them statically in the string.

Reply:
[d8d26ca](https://github.com/eclipse-edc/Connector/pull/4476/commits/d8d26ca30fb53af3ea566fbb4551a16de443d9f8)